src/tests/stream/index.test.ts

```diff
@@ -34,17 +35,13 @@ describe("Stream", () => {
         // Some tests will screw up the buffer by adding invalid data, and this is
```

Comment:
you could also just not use shared state tests because they are cringe and lead to you having to do wacky wacky wizard garbage (case in point)

Reply:
maybe